### ISSUE TITLE
500 Error on Plugin Config Page

### ISSUE THREAD

**Issue by gitbrad** [dro123/redmine_meeting_room_calendar#15]
Hello,

The plugin seem to install fine, but am seeing the following when accessing the plugin configuration page:
````
 Rendered plugins/redmine_meeting_room_calendar/app/views/meeting_room_calendar/_meeting_room_calendar_settings.html.erb (4.9ms)
Completed 500 Internal Server Error in 15ms

ActiveRecord::RecordNotFound (Couldn't find Project with 'id'=all):
  app/models/project.rb:296:in `find`
````
My environment:
RHEL7
ruby 2.0.0p598 (2014-11-13) [x86_64-linux]
Rail 4.2.0
Redmine 3.0.1

Any help appreciated!

**Comment by dro123** [comment]
Hi,

fixed with 126681b1bc3c03740a05ee2b3fc6d6d5b442dfd2 (forgot to add the file to the rails 4 commit...)

**Comment by gitbrad** [comment]
Thanks, that fixed that issue.
